api/src/main/java/org/openmrs/module/patientqueueing/api/dao/PatientQueueingDao.java

```diff
@@ -11,15 +11,25 @@
 
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
+import org.hibernate.Criteria;
+import org.hibernate.criterion.Order;
+import org.hibernate.criterion.Restrictions;
+import org.openmrs.Location;
+import org.openmrs.Patient;
+import org.openmrs.Provider;
 import org.openmrs.api.db.hibernate.DbSession;
 import org.openmrs.api.db.hibernate.DbSessionFactory;
+import org.openmrs.module.patientqueueing.model.PatientQueue;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Repository;
 
+import java.util.Date;
+import java.util.List;
+
 @Repository("patientqueueing.PatientQueueingDao")
 public class PatientQueueingDao {
 	
-	protected final Log log = LogFactory.getLog(PatientQueueingDao.class);
```

Comment:
Did you take a look at our conventions for logging? https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-loggingLogging